### PR TITLE
Add API endpoint security

### PR DESCRIPTION
**Pull request by benvogler**
This is a quick PR to lock down existing API endpoints. Please take a look through all the controller files (even those unchanged) and see if I missed anything.

### CODE REVIEW

**Comment by benvogler** [comment]
Something that just occurred to me is that we don't have a way to require email verification on endpoints currently

File: src/main/java/com/monumental/controllers/MonumentController.java [hunk 1]
```diff
@@ -104,6 +109,7 @@ public Monument updateMonument(@PathVariable("id") Integer id, @RequestBody Upda
      * @return BulkCreateResult - Object representing the results of the Bulk Monument Validate operation
      */
     @PostMapping("/api/monument/bulk/validate")
+    @PreAuthorize(Authorization.isPartnerOrResearcher)
```
Comment (njd9419) [inline src/main/java/com/monumental/controllers/MonumentController.java]: Does using this also ensure that they are authenticated?

Comment (benvogler) [reply src/main/java/com/monumental/controllers/MonumentController.java]: Yes

**Comment by njd9419** [comment]
Conflicts.

**Pull request merged**